privacyidea/lib/smsprovider/HttpSMSProvider.py

```diff
@@ -46,13 +48,26 @@
 from privacyidea.lib import _
 import requests
 from urlparse import urlparse
-
+import re
 import logging
 log = logging.getLogger(__name__)
 
 
 class HttpSMSProvider(ISMSProvider):
 
+    def _mangle_phone(self, phone, config):
```

Comment:
I believe we could make this a staticmethod?

Reply:
done